stade/urls.py

```diff
@@ -29,9 +29,10 @@
     path('admin/', admin.site.urls),
 ]
 
-if settings.DEBUG:
```

Comment:
What if `debug_toolbar` were accidentally installed in production? Why do we prefer this over `if settings.DEBUG`?

Reply:
I did it because debug is enabled in testing mode, but here we were using debug to mean development mode. 

Ideally we would have some way of determining which settings file we're using.